05_debug/task05_proc.c

```diff
@@ -0,0 +1,176 @@
+// SPDX-License-Identifier: GPL-2.0
+#include <linux/kernel.h>
+#include <linux/module.h>
+#include <linux/proc_fs.h>
+#include <linux/sched.h>
+#include <linux/slab.h>
+#include <linux/uaccess.h>
+
+MODULE_LICENSE("Dual BSD/GPL");
+MODULE_AUTHOR("Anton Kotsiubailo");
+MODULE_DESCRIPTION("Example for procfs read/write");
+MODULE_VERSION("0.1");
+
+#define MODULE_TAG "procfs_mod "
+#define PROC_DIRECTORY "task"
+#define PROC_FILENAME "buffer"
+#define BUFFER_SIZE 50
+
+static char *proc_buffer;
+static size_t proc_msg_length;
+static size_t proc_msg_read_pos;
+
+static struct proc_dir_entry *proc_dir;
+static struct proc_dir_entry *proc_file;
+
+static int proc_read(struct file *file_p, char __user *buffer, size_t length,
+		     loff_t *offset);
+
+static int proc_write(struct file *file_p, const char __user *buffer,
+			size_t length, loff_t *offset);
+
+static struct file_operations proc_fops = {
```

Comment:
checkpatch complains this struct to be const, 
modify it as follows: static const struct

Reply:
done